topics/single-cell/tutorials/scrna-data-ingest/tutorial.md

```diff
@@ -0,0 +1,881 @@
+>
+> 1. {% tool [AnnData Operations](toolshed.g2.bx.psu.edu/repos/ebi-gxa/anndata_ops/anndata_ops/1.8.1+galaxy92) %}
+> 2. Make sure you are using version **1.8.1+galaxy92** of the tool (change by clicking on {% icon tool-versions %} Versions button)
+> 3. Set the following parameters:
+>    - {% icon param-file %} In *"Input object in hdf5 AnnData format"*: `Converted EBI-compatible AnnData file`
+>    - In *"Change field names in AnnData var"*:
+>        - {% icon param-repeat %} *"Insert Change field names in AnnData var"*
+>            - *"Original name"*: `name`
+>            - *"New name"*: `Symbol`
+>    - *"Gene symbols field in AnnData"*: `Symbol`
+>    - In *"Flag genes that start with these names"*:
+>        - {% icon param-repeat %} *"Insert Flag genes that start with these names"*
+>            - *"Starts with"*: `mt-`
+>            - *"Var name"*: `mito`
+> 4. Rename {% icon galaxy-pencil %} output `AnnData for Filter, Plot, Explore workflow`
+>
+{: .hands_on}
+
+The object is ready to start its journey through the [Filter, Plot, Explore workflow](https://singlecell.usegalaxy.eu/u/j.jakiela/w/copy-of-filter-plot-and-explore-single-cell-rna-seq-data-imported-from-uploaded-file-3). All thanks to one conversion tool - how awesome that is! 
+
+
+However, sometimes it is useful to know how to do this conversion manually or at least to know how it all works. Therefore, below are some examples showing how to do it. 
+
+# AnnData -> Seurat
+
+Let's get an **AnnData object** that we can further work on. It's the object used in [many tutorials]({% link topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.md %}), so check it out if you're curious. 
+
+> <hands-on-title>Get toy data</hands-on-title>
+>
+> 1. Create a new history for this tutorial 
+> 2. Import the AnnData object from [Zenodo]({{ page.zenodo_link }})
+>
+>    If you do this tutorial just for learning purposes, you can download the downsampled dataset which will be much quicker to process:
+>    
+>    ```
+>    https://zenodo.org/record/10391629/files/Downsampled_annotated_AnnData.h5ad
+>    ```
+>    
+>    If you want to use the full dataset used in the other single-cell case study tutorials, here it is! Please note, it will take much longer to process it, so we will only show the conversions on the downsampled objects.
+>    
+>    ```
+>    https://zenodo.org/record/7053673/files/Mito-counted_AnnData
+>    ```
+> 
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 4. **Rename** {% icon galaxy-pencil %} the datasets `Downsampled AnnData object`
+> 5. Check that the datatype is `h5ad`
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="h5ad" %}
+>
+{: .hands_on}
+
+Most of our conversions involve extracting tables from different data objects and importing them into the target object.
+
+First, we will extract observations (cell metadata) and the full matrix from our AnnData.
```

Comment:
Indeed, the idea was to show the manual conversion to give the users an idea of the structure of those objects and how they are related. 
The new SCEasy converter was introduced first in this tutorial, because it's the easiest to use and probably does all the conversion the users might need. I thought it's straightforward enough to mention it at the beginning and then just show other methods, but your comment gave me an idea of adding an additional hands-on box here that will show the one-step conversion with the new SCEasy converter.